caffe2/python/convnet_benchmarks.py

```diff
@@ -85,7 +85,7 @@ def MLP(order):
     return model, d
 
 
-def AlexNet(order):
+def AlexNet(order, mkl):
```

Comment:
while MKL lives in its own folder, IMO we shouldn't make convnet_benchmark depend on it yet. How about creating a similar one in MKL's folder?